Sophos/sophos firewall/_meta/smart-descriptions.json

```
@@ -337,5 +337,19 @@
         "field": "sophos.log_type"
       }
     ]
+  },
+  {
+    "value": "{http.request.method} request by {source.ip} to {host.name} server",
```

I suggest changing the smart-description into:
```suggestion
    "value": "{source.ip} requested {url.original} on {destination.domain} server",
```